### ISSUE TITLE
e2e tests enhancements placeholder

### ISSUE THREAD

**Issue by viniarck** [kytos-ng/kytos-end-to-end-tests#155]
This is a general placeholder for general enhancements that are expected to be assessed / refined to be delivered on 2022.3:

- How can we run more tests in parallel? How to evolve the existing ones? Currently, it's taking 3 hours
- Potential retries when on tests when they first fail? What else can be done to minimize false positives? (done on https://github.com/amlight/kytos-end-to-end-tests/issues/185)
- Issue #152 

Feel free to keep complementing this issue or add topics that will be in the scope

cc'ing @italovalcy who will lead this effort

**Comment by italovalcy** [comment]
After some discussions and considering other priorities were added, we agreed to postpone the other two goals (1. running tests in parallel and 3. leveraging cookies on dump-flows to filter wanted flows) for the next release. This way, we will deliver on 2022.3 the improvements related to rerun and also improvements on the e2e tests themselves (new tests, fixes, updates, etc).
